src/ui/server/helpers/format-sort-code/index.ts

```diff
@@ -0,0 +1,28 @@
+import { DEFAULT } from '../../content-strings';
+
+const { EMPTY } = DEFAULT;
+
+export const SORT_CODE_LENGTH = 6;
```

Comment:
can this be imported from a constant?

Reply:
Good shout - now using `MAXIMUM_CHARACTERS.SORT_CODE`